server/src/main/antlr3/org/apache/iotdb/db/sql/parse/TqlLexer.g

```diff
@@ -538,3 +538,8 @@ WS  :   ( ' '
         | '\n'
         ) {$channel=HIDDEN;}
     ;
+
+Duration
+    :
+    (NUM+ ('Y'|'MO'|'W'|'D'|'H'|'M'|'S'|'MS'|'US'|'NS'))+
```

Comment:
Hi, it seems that the duration unit is case sensitive.  But is it necessary?

Reply:
Although written in uppercase, it can recognize both uppercase and lowercase.
And I found if it was written in lowercase, it doesn't recognize uppercase or lowercase.

Reply:
I'll use fragment Letter defined in TqlLexer to keep consistent with other rules.